src/lockstep/lockstep_api.py

```diff
@@ -85,33 +92,44 @@ def __init__(self, env: str):
     """
     Configure this API client to use API Key authentication
     """
+
     def with_api_key(self, apiKey: str):
         self.apiKey = apiKey
         self.bearerToken = None
-    
+
     """
     Configure this API client to use Bearer Token authentication
     """
+
     def with_bearer_token(self, bearerToken: str):
         self.apiKey = None
         self.bearerToken = bearerToken
-    
+
     """
     Send a request and parse the result
     """
+
     def send_request(self, method: str, path: str, body: object, query_params: object) -> LockstepResponse:
+
         if query_params:
             url = urllib.parse.urljoin(self.serverUrl, path) + "?" + urllib.parse.urlencode(query_params)
         else:
             url = urllib.parse.urljoin(self.serverUrl, path)
-        
+
+        headers = {"Accept": "application/json",
+                   "SdkName": self._sdkName,
+                   "SdkVersion": self._sdkVersion,
+                   "MachineName": self._machineName,
+                   "ApplicationName": ""}
+
         if self.apiKey:
-            headers = {"Accept": "application/json", "Api-Key": self.apiKey}
+            headers["Api-Key"] = self.apiKey
         elif self.bearerToken:
-            headers = {"Accept": "application/json", "Authorization": "Bearer " + self.bearerToken}
-        else:
-            headers = {"Accept": "application/json"}
-    
+            headers["Authorization"] = "Bearer " + self.bearerToken
+
+        if self._applicationName is not None:
+            headers["ApplicationName"] = self._applicationName
+
```

Comment:
Are we confident with these changes?  Shall we put them into the swagger sdk generator next?

Reply:
Added the edits now.